### DISCUSSION TITLE
Unable to delete user from Supabase UI

### DISCUSSION THREAD

**Discussion by Lalit64** [supabase/supabase#3296] [Questions] [answered]
Hi, Supabase Team. I am having trouble deleting users through the Supabase Authentication Tab. I get an error saying:

> Deleting user failed: update or delete on table "users" violates foreign key constraint "objects_owner_fkey" on table "objects"

I only have one table and it is called "profiles" and the user I am deleting doesn't have any foreign relations in that Table.

Thanks in advance, @Lalit64

**Comment by bnjmnt4n** [comment, marked as answer]
Did the user create any objects in Supabase Storage? This looks related to https://github.com/supabase/storage-api/issues/65.

**Reply by robhicks** [reply]
You need to delete the associated entry in the objects table.

**Reply by J0** [reply]
Hey all,

@bnjmnt4n  and @robhicks are right, you need to delete the associated entry in the objects table.

Thanks everyone!
